improver/utilities/cube_manipulation.py

```diff
@@ -43,6 +42,25 @@
 from improver.utilities.cube_checker import check_cube_coordinates
 
 
+def collapsed(cube, *args, **kwargs):
+    """Collapses the cube with given arguments but doesn't keep the cell method
+
+    Args:
+        cube (iris.cube.Cube):
+            A Cube to be collapsed.
+        *args:
+        **kwargs:
```

Comment:
Do these need to be included in the doc-string, given that they can be anything?

Reply:
Fair, fixed.